src/states/michigan/main.py

```diff
@@ -15,7 +15,8 @@
 SSL_CERT = os.path.join(dir_path(__file__), 'michigan_chain.pem')
 
 re_official = re.compile(r'^\s*(.*?)\s*[,\n]')
-re_address = re.compile(r'\n(.*)\nPhone', flags=re.MULTILINE + re.DOTALL)
+re_phys_addr = re.compile(r'\n(.*?\d{5}(?:-\d{4})?)\n', flags=re.MULTILINE + re.DOTALL)
```

Comment:
NIT: I think the convention is to use `re.MULTILINE | re.DOTALL`.

https://stackoverflow.com/questions/13110907/use-of-re-multiline-and-re-dotall-together-python